src/controllers/handlers/scene-admin-handlers/add-scene-admin-handler.ts

```diff
@@ -0,0 +1,65 @@
+import { EthAddress } from '@dcl/schemas'
+import { InvalidRequestError, UnauthorizedError } from '../../../types'
+import { HandlerContextWithPath } from '../../../types'
+import { validate } from '../utils'
+
+export async function addSceneAdminHandler(
+  ctx: Pick<
+    HandlerContextWithPath<'fetch' | 'sceneAdminManager' | 'sceneFetcher' | 'logs' | 'config', '/scene-admin'>,
+    'components' | 'request' | 'verification' | 'url' | 'params'
+  >
+) {
+  const {
+    components: { logs, sceneAdminManager, sceneFetcher },
+    request,
+    verification
+  } = ctx
+
+  const { getPlace, hasLandPermission, hasWorldPermission } = sceneFetcher
+
+  const logger = logs.getLogger('add-scene-admin-handler')
+
+  if (!verification?.auth) {
+    throw new InvalidRequestError('Authentication required')
+  }
+
+  const payload = await request.json()
+  if (!payload.admin || !EthAddress.validate(payload.admin)) {
+    logger.warn(`Invalid scene admin payload`, payload)
+    throw new InvalidRequestError(`Invalid payload`)
+  }
+
+  const { parcel, hostname, realmName } = await validate(ctx)
+  const isWorlds = hostname!.includes('worlds-content-server')
+  const authAddress = verification.auth
```

Comment:
I'm not sure if this works to extract signing address from the auth-chain 🤔, also, if it is the right way, should we call EthAddress.validate with it?

Reply:
we are using `EthAddress.validate` to validate the payload.